README.md

```diff
@@ -1 +1,17 @@
-# open-api-sample-app
\ No newline at end of file
+# Sample Application using OpenAPI Client Library.
+
+> Please note that this is work in progress!
+
+This repository contains sample applications, which shows how to consume Saxo Bank's OpenAPIs using open source library.
+This is little advance sample, Please go through our [Foundational Samples](FoundationalSamples) to understand OpenAPI basics.
+
+Target audience are technical people and client developers.
+
+Please go through our open source library to know more about OpenAPI client Library.
+
+
+## Prerequisite
+To use this sample you need a valid access token from [the developer portal](https://developer.saxobank.com/sim/openapi/portal/)
+
```

Comment:
Another prerequisite is to set up the node environment an load node modules using 
npm install 
To run the project we need to do 
node server.js